packages/commonwealth/client/scripts/controllers/app/webWallets/near_web_wallet.ts

```diff
@@ -35,6 +35,9 @@ class NearWebWalletController implements IWebWallet<any> {
   public get enabled() {
     return this._enabled;
   }
+
+  api: () => any
```

Comment:
Remove these -- doesn't make sense anyway because `any` is a type, not a value.

Reply:
whoops forgot about leaving these in there my bad